app/routes.js

```diff
@@ -88,7 +88,7 @@ const stripeSetupDashboardRedirectController = require('./controllers/stripe-set
 const {
   healthcheck, registerUser, user, dashboard, selfCreateService, transactions, credentials,
   apiKeys, serviceSwitcher, teamMembers, staticPaths, inviteValidation, editServiceName, merchantDetails,
-  notificationCredentials: nc, paymentTypes: pt, emailNotifications: en, toggle3ds: t3ds, toggleMotoMaskCardNumberAndSecurityCode, prototyping, paymentLinks,
+  notificationCredentials: nc, paymentTypes: pt, emailNotifications: en, toggle3ds, toggleMotoMaskCardNumberAndSecurityCode, prototyping, paymentLinks,
```

Comment:
nice cleanup 👍 . It would be good if this could also move under `paths.account` to make it clear it's now using the new structure and allow any code that uses that group to include this route

Reply:
done